lib/account/account-manager.js

```diff
@@ -5,6 +5,23 @@
  */
 
 /**
+ * Information about an account holder.
+ *
+ * @typedef {Object} accountManager~accountFile
+ * @property {string} email
+ * @property {string} mfaKey
+ * @property {string} secureHash
+ * @property {string} passwordSalt
+ */
+
+/**
+ *
```

Comment:
A better description will help me understand when this is used.  Not necessarily saying "this is returned by getThingAsync" but more along the lines of "This collection of information is sent back to the client so they are able to generate password hashes" or something.